src/main/java/twilightforest/gt_integration/GT_OrePlacer.java

```diff
@@ -0,0 +1,54 @@
+package twilightforest.gt_integration;
+
+import net.minecraft.block.Block;
+import net.minecraft.init.Blocks;
+import net.minecraft.world.World;
+
+import gregtech.common.blocks.GT_TileEntity_Ores;
+import twilightforest.TwilightForestMod;
+
+public class GT_OrePlacer {
+
+    public int getMappedGTMetaForOre(Block mOre) {
```

Comment:
`getMappedGTMetaForOre` should be private, and if you remove `GT_Integration_Utils::getOrePlacer` then all of GT_OrePlacer can be package private instead of public to hide your implementation details.